Storage/SQL/SQLiteBookmarksSyncing.swift

```diff
@@ -438,8 +438,11 @@ public class SQLiteBookmarkBufferStorage: BookmarkBufferStorage {
             "folderName = ?, queryId = ? " +
             "WHERE guid = ?"
 
+            // We used to use INSERT OR IGNORE here, but it muffles legitimate errors. The only
+            // real use for that is/was to catch duplicates, but the UPDATE we run first should
+            // serve that purpose just as well.
             let insert =
-            "INSERT OR IGNORE INTO \(TableBookmarksBuffer) " +
```

Comment:
So now if we run into the case where we would have IGNORE'ed, we explicitly fail instead, correct?